pkg/tsdb/elasticsearch/time_series_query.go

```diff
@@ -454,7 +454,16 @@ func (p *timeSeriesQueryParser) parseMetrics(model *simplejson.Json) ([]*MetricA
 		metric.Hide = metricJSON.Get("hide").MustBool(false)
 		metric.ID = metricJSON.Get("id").MustString()
 		metric.PipelineAggregate = metricJSON.Get("pipelineAgg").MustString()
-		metric.Settings = simplejson.NewFromAny(metricJSON.Get("settings").MustMap())
+		// In legacy editors, we were storing empty settings values as null
```

Comment:
nitpick: i think we should emphasize in the comment that this is a string with value `"null"`, not a json-null.
```suggestion
		// In legacy editors, we were storing empty settings values as "null"
```